File: list_freezones.py
import json
import subprocess
import sys

try:
    output = subprocess.check_output(['curl', '-X', 'GET', 'http://localhost:5000/api/free-zones'])
    freezones = json.loads(output)
    
    print("Available Free Zones:")
    print("---------------------")
    for zone in freezones:
        print(f"ID: {zone['id']} - Name: {zone['name']}")
    
except Exception as e:
    print(f"Error: {e}")
    sys.exit(1)

File: list-freezones.ts
import { db } from './server/db';
import { freeZones } from './shared/schema';

async function listFreeZones() {
  try {
    const zones = await db.select().from(freeZones);
    
    console.log(`Total free zones in database: ${zones.length}`);
    console.log('\nList of free zones:');
    zones.forEach((zone, i) => console.log(`${i+1}. ${zone.name}`));
    
  } catch (err) {
    console.error('Error fetching free zones:', err);
  }
}

listFreeZones().catch(console.error);